pyproject.toml

```diff
@@ -39,8 +39,7 @@ ignore = [
 minversion = "6.0"
 addopts = "--ignore=setup.py --ignore=validation/ --ignore=binder/ --ignore=docs/ --cov=pyhf --cov-report=term-missing --cov-config=.coveragerc --cov-report xml --doctest-modules --doctest-glob='*.rst'"
 testpaths = [
-    "src",
-    "tests",
+    "tests"
```

Comment:
We're not not having the docstrings get checked by `doctest` if this is removed. So we'll have to fix ths.

Reply:
@kratsg You [say that for `ironman`](https://twitter.com/kratsg/status/1394483751758008323)

> Dropping "src" from "testpaths" fixes it for me and still runs doctest correctly.

but I'm not really clear on how this works then for `doctest`. Can you take a look here as well?